src/Nethermind/Nethermind.Merge.Plugin/Handlers/GetPayloadBodiesByRangeV1Handler.cs

```diff
@@ -23,22 +24,45 @@ public GetPayloadBodiesByRangeV1Handler(IBlockTree blockTree, ILogManager logMan
         _logger = logManager.GetClassLogger();
     }
 
-    public Task<ResultWrapper<ExecutionPayloadBodyV1Result?[]>> Handle(long start, long count)
+    public Task<ResultWrapper<IEnumerable<ExecutionPayloadBodyV1Result?>>> Handle(long start, long count)
     {
         if (count > MaxPayloadBodies)
```

Comment:
We support limits for getPayloadsByRange but not for getPayloadsByHashes. After commenting on this "if" I'm still passing the tests -> missing unit tests scenarios